### PR TITLE
test(e2e): add mayastor pod disruption test

### PR DESCRIPTION
**Pull request by chriswldenyer**
Re-introduce mayastor pod-suppression disruptive test

Rename and relocate some source files.
Add the mayastor pod disruptive test to the continuous and nightly
test runs.
Generate Slack alerts from "continuous" only in the develop branch.
Update continuous tested version to 0.8.0
To do in next PRs: 
* use new run-on-deploy e2e_fio image
* migrate faulted replica a second time.
* ensure the original copy no longer exists
* run continuous against "latest" but obtain the corresponding
version number

### CODE REVIEW

**Comment by chriswldenyer** [comment]
bors try

**Comment by chriswldenyer** [comment]
bors try

**Comment by chriswldenyer** [comment]
bors try

**Comment by chriswldenyer** [comment]
bors try

**Review by blaisedias — APPROVED** [review]
Look okay to me.

File: test/e2e/ms_pod_disruption/lib/disruption_lib.go [hunk 1]
```diff
@@ -14,9 +14,13 @@ const (
 	podUnscheduleTimeoutSecs = 100
 	podRescheduleTimeoutSecs = 180
 	repairTimeoutSecs        = "180s"
+	mayastorRegexp           = "^mayastor-.....$"
+	moacRegexp               = "^moac-..........-.....$"
+	engineLabel              = "openebs.io/engine"
```
[the remark this answers is not in the document]
Comment (chriswldenyer) [reply test/e2e/ms_pod_disruption/lib/disruption_lib.go:19]: Yes, I think so. I'll move it there next iteration.

**Review by paulyoong — APPROVED** [review]

**Comment by chriswldenyer** [comment]
bors merge

**Pull request merged**